_drafts/gnu-make-guidelines.md

```diff
@@ -0,0 +1,831 @@
+---
+title: "GNU Make Overview and Guidelines"
+description: "A brief overview of GNU make features and suggestions for getting
+the most out of it"
+author: noah
+image: /img/gnu-make-guidelines/gnu-make-guidelines.img
+---
+
+GNU Make is a very commonly used program for building C language software. It's
```

Comment:
GNU Make is a commonly used program and build system for building C/C++ language software. It is the build system of choice for the [Linux kernel](https://wiki.archlinux.org/index.php/Kernel/Traditional_compilation#Compilation),
and many other commonly used GNU/Linux programs and software libraries.